tests/vmi_configuration_test.go

```diff
@@ -287,6 +289,30 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			),
 		)
 
+		It("with downwardMetrics virtio-serial", func() {
+
+			vmi := libvmi.New(
+				libvmi.WithDownwardMetricsChannel(),
+				libvmi.WithResourceMemory("1Gi"),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			)
+
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 60)
+			libwait.WaitForSuccessfulVMIStart(vmi)
+			expectedXML := fmt.Sprintf("<source mode='bind' path='%s'/>", downwardmetrics.DownwardMetricsChannelSocket)
+
+			domXml, err := tests.GetRunningVirtualMachineInstanceDomainXML(virtClient, vmi)
```

Comment:
Do we have a better method of validating the XML instead of just searching for substrings?

Reply:
Yes, we have implemented a fancier way to do this.